webapp/channels/src/components/suggestion/command_provider/command_provider.tsx

```diff
@@ -209,7 +209,7 @@ export default class CommandProvider extends Provider {
     }
 
     handleWebapp(pretext: string, resultCallback: ResultsCallback<AutocompleteSuggestion>) {
-        const command = pretext.toLowerCase();
+        const command = pretext;
```

Comment:
Looking at the server side, this is expected to be lower cased, so it would also create problems similar to my previous comment.